python/doc/examples/meta_modeling/chaos_ishigami.ipynb

```diff
@@ -63,8 +63,13 @@
     "ST_1 = \\frac{V_1+V_{1,3}}{V(Y)}, \\qquad ST_2 = S_2, \\qquad S_3 = \\frac{V_{1,3}}{V(Y)}.\n",
     "$$\n",
     "\n",
-    "The third variable $X_3$ has no effect at first order (because $X_3^4$ it is multiplied by $\\sin(X_1)$), but has a total effet because of the interactions with $X_1$. On the other hand, the second variable $X_2$ has no interactions which implies that the first order indice is equal to the total order indice for this input variable.\n",
-    "\n",
+    "The third variable $X_3$ has no effect at first order (because $X_3^4$ it is multiplied by $\\sin(X_1)$), but has a total effet because of the interactions with $X_1$. On the other hand, the second variable $X_2$ has no interactions which implies that the first order indice is equal to the total order indice for this input variable."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
```

Comment:
Not on this line, but below:
We could use only the input and output training samples: in this case, the distribution of the input sample is computed by selecting the best distribution that fits the data.
We could use only the input and output training samples: in this case, the distribution of the input sample is computed by selecting the distribution that has the best fit.

Reply:
Fixed.